Credify/Credify/WebView/WebPresenter.swift

```diff
@@ -337,6 +357,35 @@ class WebPresenter: WebPresenterProtocol {
         }
     }
     
+    func goToPreviousPageOrClose(webView: WKWebView) {
+        let url = webView.url?.absoluteString
+        isLoading(webView: webView) { isLoading in
+            if isLoading {
+                return
+            }
+            
+            if !webView.canGoBack {
+                self.hanldeCompletionHandler()
+                return
+            }
+            
+            switch self.context {
+            case .mypage(_), .serviceInstance:
+                if url != nil && (url!.starts(with: self.pendingUrl) || url!.starts(with: self.canceledUrl)) {
```

Comment:
There are too many logics about URL checking. You should centralize the URL related logic to one single place. You can make URLhelper or something like that and move the logic to this file.

Reply:
And move `pendingUrl` too

Reply:
I updated my code too